core/trino-main/src/main/resources/oauth2/success.html

```diff
@@ -44,7 +44,7 @@
     <link href="/ui/vendor/css-loaders/loader.css" rel="stylesheet">
 
     <!-- Custom CSS -->
-    <link href="/ui/assets/presto.css" rel="stylesheet">
+    <link href="/ui/assets/trino.css" rel="stylesheet">
```

Comment:
don't you need to rename some file?

Reply:
there is no `presto.css` file, `trino.css` is already used in various HTMLs like oauth2 `failure.html`

Reply:
Oh... so it is actual separate issue then. One is to change the content and other one is to fix the referenced file for css. Please do two commits for this.